tests/framework/population/conftest.py

```diff
@@ -92,3 +92,52 @@ def _add_constraint(self, *args: Any, **kwargs: Any) -> None:
         "cube_component": CUBE_COL_NAMES,
     }
     return mgr
+
+
+# Helper assertion functions for testing squeezing behavior
```

Comment:
Shouldn't these be in a testing utilities module, rather than the conftest? I thought the conftest was for fixtures.